visual-js/.changeset/hot-stingrays-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+"@saucelabs/nightwatch-sauce-visual-service": minor
```

Comment:
From what I've tested with all our examples it's backwards compatible, but since it's a fairly major refactor for bundling & deployment I feel safer pushing it into a `major` version so it's not automatically bumped